pinot-filesystem/src/main/java/com/linkedin/pinot/filesystem/LocalPinotFS.java

```diff
@@ -133,11 +141,25 @@ public boolean isDirectory(URI uri) throws IOException {
     return new File(uri).isDirectory();
   }
 
-  private String encodeURI(String uri) throws UnsupportedEncodingException {
-    return URLEncoder.encode(uri, DEFAULT_ENCODING);
+  private String encodeURI(String uri) {
```

Comment:
Why changing this?

Reply:
I don't want to unnecessarily have to do a bunch of try/catch statements in the code for pinotfs apis that shouldn't throw IOExceptions (eg: isDirectory)

Reply:
If you revert this, I think it still works.